lib/shopify-cli/core/update.rb

```diff
@@ -0,0 +1,113 @@
+require 'shopify_cli'
+
+module ShopifyCli
+  module Core
+    module Update
+      FETCH_HEAD = File.expand_path('.git/FETCH_HEAD', ShopifyCli::ROOT)
+
+      class << self
+        def auto_update
+          return if Context.new.testing?
+          prompt_for_updates
+          return unless ShopifyCli::Config.get_bool('autoupdate', 'enabled')
+
+          # don't update more than once per hour.
+          file = begin
+                  File.mtime(FETCH_HEAD)
+                 rescue Errno::ENOENT
+                   Time.at(0)
+                end
+          age = Time.now - file
+          return unless age > 3600
+
+          check_now(restart_command_after_update: true)
+        end
+
+        def check_now(restart_command_after_update: raise, ctx: ShopifyCli::Context.new)
+          if Context.new.development?
+            if restart_command_after_update
+              return # just skip
+            else
+              err("Development version of {{cyan:shopify}} in use. Run {{cyan:shopify load-system}} first.")
```

Comment:
Should references to `{{cyan:shopify}}` be changed to `{{command:#{ShopifyCli::TOOL_NAME}}}` throughout this file for consistency with, for example, the content of the `help()` methods?

Reply:
Yes, definitely. I'll update those. Great catch

Reply:
Updated all the existing ones now (except for the ones you're updating in your `populate` PR so hopefully we'll never have this problem again lol
If you want to take another 👀 and  ✅